packages/drivers/routerlicious-driver/src/shreddedSummaryDocumentStorageService.ts

```diff
@@ -98,13 +98,14 @@ export class ShreddedSummaryDocumentStorageService implements IDocumentStorageSe
 
 	public async getSnapshotTree(version?: IVersion): Promise<ISnapshotTreeEx | null> {
 		let requestVersion = version;
-		if (!requestVersion) {
+		if (requestVersion === undefined) {
```

Comment:
There used to be calls to this method that used `null`. Probably not anymore, but might be worth double checking if you haven't already